src/app/home/home.component.html

```diff
@@ -14,4 +14,9 @@
       <a routerLink="/treelist" routerLinkActive="active">Tree List</a>
     </div>
   </div>
+  <div class="row">
+    <div class="col-xs-12">
```

Comment:
is this verified to work at the `xs` size? if not, please set to `col-sm-12` to match the other widgets.